tests/ui/vec.rs

```diff
@@ -22,28 +27,38 @@ impl Line {
 fn main() {
     on_slice(&vec![]);
     on_slice(&[]);
+    on_mut_slice(&mut vec![]);
 
     on_slice(&vec![1, 2]);
     on_slice(&[1, 2]);
+    on_mut_slice(&mut vec![1, 2]);
 
     on_slice(&vec![1, 2]);
     on_slice(&[1, 2]);
+    on_mut_slice(&mut vec![1, 2]);
     #[rustfmt::skip]
-    on_slice(&vec!(1, 2));
+        on_slice(&vec!(1, 2));
```

Comment:
Is this spaces what you intended?

Reply:
ah sorry, didn't realise that and the skip flag was of course skipping this.
There was no particular reason for those spaces; I just pushed again without them.
thanks for the patience.